explorer/explorerroutes.go

```diff
@@ -337,7 +337,7 @@ func (exp *explorerUI) NextHome(w http.ResponseWriter, r *http.Request) {
 	exp.pageData.RLock()
 	mempoolInfo.Subsidy = exp.pageData.HomeInfo.NBlockSubsidy
 
-	str, err := exp.templates.exec("nexthome", struct {
+	str, err := exp.templates.execTemplateToString("visualblocks", struct {
```

Comment:
We're using the new `exec` method now. Changed in #1505.

Reply:
bumpy bump, @codemaestro64